external/netty/build.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
```

Comment:
If we auto-generate the dockerfiles, the build.xml file can be simplified, you can use [derby build.xml](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/external/derby/build.xml) file as an example